test/test_filesystem_helper.cpp

```diff
@@ -392,3 +392,15 @@ TEST(TestFilesystemHelper, get_cwd)
   auto p = rcpputils::fs::current_path();
   EXPECT_EQ(expected_dir, p.string());
 }
+
+TEST(TestFilesystemHelper, parent_absolute_path)
+{
+  rcpputils::fs::path path("/home/foo/bar/baz");
+  ASSERT_EQ(path.string(), "/home/foo/bar/baz");
```

Comment:
This new test (and potentially the ones after it) fail Windows CI:

```
[ RUN      ] TestFilesystemHelper.parent_absolute_path
C:\ci\ws\src\ros2\rcpputils\test\test_filesystem_helper.cpp(399): error: Expected equality of these values:
  path.string()
    Which is: "\\home\\foo\\bar\\baz"
  "/home/foo/bar/baz"
[  FAILED  ] TestFilesystemHelper.parent_absolute_path (0 ms)
```

Reply:
It would be Windows, wouldn't it?  Thanks for the heads up, I'll fix it.